src/pages/signUpView.js

```diff
@@ -63,6 +64,7 @@ export default function SignUpView() {
     }
 
     if (validSignup) {
+      setSuccessfulSignUp(true);
```

Comment:
It's much less likely to happen on sign up, but it's also possible to be locked out here. Similarly to the login page, running `setSuccessfulSignUp(false)` on line 82 should fix that.